docs/source/qvm.rst

```diff
@@ -419,8 +419,8 @@ Simulating the QPU using the QVM
 The :py:class:`~pyquil.api.QAM` methods are intended to be used in the same way, whether a QVM or QPU is being targeted.
 Everywhere on this page,
 you can swap out the type of the QAM (QVM <=> QPU) and you will still
-get reasonable results back. As long as the topology of the devices are the same, programs compiled and ran on the QVM
-will be able to run on the QPU and visa-versa. Since :py:class:`~pyquil.api.QuantumComputer` is built on the ``QAM``
+get reasonable results back. As long as the topology of the devices are the same, programs compiled and run on the QVM
```

Comment:
> visa-versa

This pronunciation is a pet-peeve of mine. I didn't realise people actually wrote it this way too :D

Reply:
I can't quite get the grammar right. Should it be

> As long as the topology of the devices is the same, ...

or

> As long as the topologies of the devices are the same, ...

or

> ???

Reply:
I think either is acceptable in this case? Maybe because "topology" is an abstract concept, and we are specifically claiming that they "are the same", hence there is really only one topology?

It's like if I said: "The number of cheetos in all the bags is the same."

But I am only a Junior Grammar Nazi in Training, and not 100% sure about this.

Reply:
“Topologies” sounds most natural to me, but I am an outlier in how tangible I consider a topology to be.

Reply:
I just realized that the text in the original is "topology .... are" which I'm fairly certain is wrong. Verb and noun should at least agree, which I guess was @notmgsk's point.

I trust @ecpeterson's instincts here, so "topologies" it is!

Reply:
Done.